### DISCUSSION TITLE
Is Ftxf25 - Siesta supported?

### DISCUSSION THREAD

**Discussion by francescocam** [revk/xESP32-Faikout#165] [General]
Hi,
I have this split. Unfortunately the wifi module is the one that connects to the onecta app.
I took a look at the board and I am not sure that if it has a compatible port.
I am attaching a picture of the board hoping for some help.
![20231213_080014](https://github.com/revk/ESP32-Faikin/assets/1070900/8d2fed53-28f8-43b5-9147-be7074809d1d)

**Comment by xudabit** [comment]
Hi,

you have a S403-Port, which I'am using with my ATXDs (also Siesta). It's working good so far, but you need be really careful with the pin on the right, the one that's standing alone. This one has more than 200VDC and should neither be touched nor connected to anything.
